app/assets/javascripts/delivery_orders.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Please remove this unused file if the checkFile method will be continue in the view file. if not put method here